package/correct_live_install

```diff
@@ -0,0 +1,34 @@
+#! /bin/sh
+
+# this script is called from the live installer once it is done (including 2nd stage)
+# So the goal of this script is to fix whatever configs were changed for the live system
+
+#======================================
+# /etc/sudoers hack to fix #297695 
+# (Installation Live CD: no need to ask for password of root)
```

Comment:
in kiwi-config-openSUSE package, the scripts went in kiwi image description root/usr/bin/